res/schema.xml

```diff
@@ -582,4 +582,28 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Suggestion tables added.
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS suggestion_library (
+        id INTEGER,
+        artist varchar(48),
+        title varchar(48),
+        playcount INTEGER,
+        match FLOAT,
+        duration FLOAT,
+        location varchar(512));
+    </sql>
+  </revision>
+    <revision version="41" min_compatible="3">
+    <description>
+      Playcount and match column added to library table for sorting.
+    </description>
+    <sql>
+      ALTER TABLE library ADD COLUMN playcount float DEFAULT NULL;
```

Comment:
There is already `timesplayed` with a non-trivial synchronization mechanism to keep it in sync with the history playlists. What is this new column for?

Reply:
If it is some external play count then it doesn't belong here anyway.

Reply:
It is the play count of last.fm. It can be used to compare tracks for popularity. I have already suggested to rename this column to make it clear.

Reply:
last.fm metadata should be stored separately. The same case as with any other external library integration.

Reply:
Yes, that's right, this dies not at all belongs to the library table. This data makes only sense for the "suggestion_library" where the relation of two tracks can be expressed.